gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -209,16 +208,20 @@ const postChangeUnissuedFacility = async (req, res) => {
     req.success = {
       message: `${body.facilityName} is updated`,
     };
-    req.url = `/gef/application-details/${dealId}/unissued-facilities`;
 
     // updates application with editorId
     const applicationUpdate = {
       editorId,
     };
     await api.updateApplication({ dealId, application: applicationUpdate, userToken });
 
-    // TODO: DTFS2-5227 change redirect
-    return applicationDetails(req, res);
+    req.flash('success', {
+      message: `${body.facilityName} is updated`,
+    });
+
+    const redirectUrl = `/gef/application-details/${dealId}/unissued-facilities`;
```

Comment:
Before consuming `dealId` please verify dealId is a valid deal ID, implementation already exists in the codebase.